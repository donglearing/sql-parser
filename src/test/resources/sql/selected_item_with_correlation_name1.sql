select t.a as ta from T as t;