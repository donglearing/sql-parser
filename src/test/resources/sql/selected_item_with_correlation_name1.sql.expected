SELECT t.a AS ta FROM T AS t